Applications/CMC/test/testCMCEMGDrivenArm_Thelen.cpp

```diff
@@ -42,17 +42,17 @@ void testCMCEMGDrivenArm() {
     Storage *standard = new Storage();
     cmc.getModel().formStateStorage(temp, *standard);
 
-    Array<double> rms_tols(0.02, 2*2+2*6);
```

Comment:
Was it necessary to loosen the tolerances on the non-activation states (angles, speeds, and fiber-lengths) to get the test to pass?

The activation are being forced to change because a made up "EMG" signal is being used to set the target of the activation state, but the tracking of the coordinates and speeds should still be tight.